algo-core/ortools_solver.py

```diff
@@ -41,14 +41,16 @@
 # Other constants:
 max_avg_per_week = 40
 week_working_hours = 40
-slots_in_day = 24
+slots_in_day = 26
 date_format = "%Y-%m-%d"
 
 # Input filenames:
 filename_onboarding = "onboarding_agents.txt"
 filename_mentors = "mentors.txt"
 filename_new = "new_agents.txt"
 
+def get_project_root() -> Path:
```

Comment:
I think we could improve this reliance on relative paths. This feels error prone to me. No need to address it in this PR.